tools/Utilities/src/Strings/en-us/Resources.resw

```diff
@@ -118,14 +118,13 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="EnvVariablesEditorUtilityDesc" xml:space="preserve">
-    <value>A quick utility for managing environment variables.</value>
+    <value>Create and manage variable sets with a lightweight UI.</value>

```

Comment:
I'm curious.  What is a lightweight UI?

Reply:
I know content design has had a pass at these, but it does seem better to just leave off the lightweight UI comment :)

Reply:
"Utility to create and manage variable sets" ?

cc: @cinnamon-msft

Reply:
Updated.